cucumber-selenium-tests/src/test/resources/com/skillstorm/tax-page.feature

```diff
@@ -0,0 +1,47 @@
+Feature: update tax estimation form
+
+
+  Scenario Outline: view existing tax estimation form
+    Given I am on the "<page>" page
+    Then I should see the existing tax estimation form
+
+    
+    Examples:
+    | page        |
+    | Tax  |
+
+
+  Scenario Outline: Successfully update tax estimation form
+    Given I am on the "<page>" page
+    When I click the "<button>" button 
+    And i enter the following personal information "<firstName>" "<lastName>" "<streetAddress>" "<city>" "<state>" "<zip>" "<DOB>" "<SSN>"
+    And I click the "<button2>" button followed by the "<button3>" button
+    And I click the "<button4>" button followed by the "<button5>" button 
+    And I enter the following W2 information "<state2>" "<employer>" "<wages>" "<federalTax>" "<stateTax>" "<SSNTax>" "<medicareTax>"
+    And I click the "<button6>" button followed by the "<button7>" button 
+    And I enter valid other income information such as "<longTermCapitalGains>" "<shortTermCapitalGains>" "<netBusinessIncome>" "<otherInvestments>" "<additionalIncome>"
+    And I click the "<button8>" button followed by the "<button9>" button 
+    And I click the "<button10>" button followed by the "<button11>" button 
+    And I enter valid deductions information such as "<deductionType>" "<deductionAmount>"
+    And I click the "<button12>" button followed by the "<button13>" button 
+    And I review and and click the "<button14>" button
+    Then I should see the updated tax estimation
+    
+    
+    
+    
+
+  Examples:
```

Comment:
Please format this table for better readability (on VSCode, right click this file while opened, and click 'Format Document'

Reply:
I right clicked and formatted the file, it still runs off to the right but it lined up the variables